Header header
float64 Latitude
float64 Longitude
float64 Altitude
float64 HDOP
float64 UTM_northing
float64 UTM_easting
float64 UTC 
int64 Zone
string Letter
int64 FIX
